dts/bindings/sensor/adi,adxl345-common.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2020 Antmicro <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: ADXL345 3-axis I2C accelerometer
+
+include: [sensor-device.yaml]
+
+properties:
+  frequency:
+    type: int
+    default: 100
+    description: |
+          Accelerometer sampling frequency (ODR). Default is 100Hz.
```

Comment:
Need to justify the default value in the description. Is it a register reset value?

Reply:
Added justification.